packages/modules/manage-versions/src/components/InformationPanel/ChangesetInformationPanel.tsx

```diff
@@ -0,0 +1,80 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import "./ChangesetInformationPanel.scss";
+
+import {
+  InformationPanel,
+  InformationPanelBody,
+  InformationPanelHeader,
+  Text,
+} from "@itwin/itwinui-react";
+import React from "react";
+
+import { Changeset } from "../../models/changeset";
+import { dateTimeFormatOptions, getLocale } from "../../models/utils";
+import { informationPanelStringOverrides } from "../ManageVersions/types";
+
+export interface ChangesetInfoPanelProps {
+  changeset: Changeset;
+  onClose: (e: React.MouseEvent<Element, MouseEvent>) => void;
+  stringOverrides: informationPanelStringOverrides;
+}
+
+export const ChangesetInformationPanel = (props: ChangesetInfoPanelProps) => {
+  const { changeset, onClose, stringOverrides } = props;
+  const files: string[] = changeset.synchronizationInfo?.changedFiles
+    ? changeset.synchronizationInfo.changedFiles
+    : [stringOverrides.noValue];
+
+  const createdDateTime = new Date(changeset.pushDateTime);
+  const formattedDate = createdDateTime.toLocaleDateString(
```

Comment:
have a function and move it outside component to get the formatted date.. Can be in same file, just above

Reply:
created function in util